src/Rings/mpoly-local.jl

```diff
@@ -166,13 +186,33 @@ end
 # Constructors for ideals                                                     #
 ###############################################################################
 
+@Markdown.doc """
+    function singular_ring_loc(R::MPolyRingLoc{T}; ord::Symbol = :negdegrevlex) where T
```

Comment:
```suggestion
   singular_ring_loc(R::MPolyRingLoc{T}; ord::Symbol = :negdegrevlex) where T
```
and more

Reply:
Ok, thx. This was not my file, but Raul's. I just added some docstrings when reading and learning it. Then it disappeared from my radar as a file that had been edited. It's now repaired.